templates/contribute/tags_and_branches.md

```diff
@@ -87,7 +108,10 @@ Mathlib.
     * This is required for Mathlib's `.olean` caching mechanism.
```

Comment:
Comment for the line above this: Now that we suggest to open branches in Std itself, "PRs to Mathlib should be made from branches of the Mathlib repository itself" is not different from Std anymore.

Reply:
Ah, it is: *all* PRs to mathlib should be made in the main repo. It's only `lean-pr-testing-NNNN` branches at Std that I'd like to be on the main repo. I've added "All".